solar_panel_connector/README.md

```diff
@@ -0,0 +1,8 @@
+# solar-panel-connector
+There are a set of solar panels housed SIT (Singapore Institute of Technology @ Nanyang Polytechnic). All raw data are overseed by their managment hub (Laurel). The research engineer there have created a MQTT Broker and topic for us to subscribe.  
+
+# How does it work?
+The set of solar panels in SIT have been equipped with multiple sensors to detect current, power, etc for monitoring purposes. These sensors will feed the data to Laurel in a .json format. The data are further broke down and are housed in a database at SIT. Selected data will be fed to MQTT broker by publishing topics. A MQTT client is set up to subscribe to these topics to obtain raw data every minute.
+
+# Current development
```

Comment:
Could you please describe which information we receiving right now (I mean what we have in that raw data)?

Reply:
Got it. Do you need me to go in-depth on what's power, current and voltage?

Reply:
No, no need to add general details
It could be a good idea to add an example of data which we receiving right now (describe what it has, just a few words)

Reply:
Okays. I've added in some details. Let me know your thoughts :)